api/helpers/things.js

```diff
@@ -95,14 +95,15 @@ const returnByType = (res, params, article, static, user) => {
 };
 
 const parseGetParams = function(req, type) {
-  return Object.assign({}, req.query, {
+  let parms = Object.assign({}, req.query, {
```

Comment:
/s/parms/params

Reply:
Ah, that was supposed to be temporary, good eye.